hetrixtools_agent.sh

```diff
@@ -165,10 +165,10 @@ if [ ! -z "$ConnectionPorts" ]
 then
 	IFS=',' read -r -a ConnectionPortsArray <<< "$ConnectionPorts"
 	declare -A Connections
-	netstat=$(netstat -ntu)
+	netstat=$(netstat -ntu | awk '{print $4}')
 	for cPort in "${ConnectionPortsArray[@]}"
 	do
-		Connections[$cPort]=$(echo "$netstat" | grep -w "$cPort" | wc -l)
+		Connections[$cPort]=$(echo "$netstat" | grep ":$cPort" | wc -l)
```

Comment:
```suggestion
		Connections[$cPort]=$(echo "$netstat" | grep ":$cPort$" | wc -l)
```

Should include `$` to avoid cases like for example: port 33 would match IP:3306 lines.